src/test/java/org/benetech/servicenet/web/rest/ServiceTaxonomyResourceIntTest.java

```diff
@@ -48,6 +49,9 @@
     private static final String DEFAULT_TAXONOMY_DETAILS = "AAAAAAAAAA";
     private static final String UPDATED_TAXONOMY_DETAILS = "BBBBBBBBBB";
 
+    @Autowired
+    private HibernateInterceptor hibernateInterceptor;
```

Comment:
why do we have to disable it before every test?

Reply:
Because all metadata is connected to the currently logged user, and it would throw an error if no user is signed in.

Reply:
I was also thinking about authorizing user before each test, but I hadn't found a solution that would work.

Reply:
Implement a test user service that extends the regular user service, but overrides the method that you are using to get the user. Then add an additional configuration bean to your test classes. In that configuration redefine the userService bean with a you new test implementation.